mlrun/model_monitoring/helpers.py

```diff
@@ -41,19 +42,22 @@ class _MLRunNoRunsFoundError(Exception):
     pass
 
 
-def get_stream_path(project: str = None, application_name: str = None):
+def get_stream_path(
+    project: str = None,
+    application_name: str = mm_constants.MonitoringFunctionNames.STREAM,
```

Comment:
See that previous comment about the naming of this argument.
Also, what happens if users try to create a model monitoring function named `mm_constants.MonitoringFunctionNames.STREAM`?

Reply:
@jond01 I think we have also a problem with name of the  controller and writer.
I can restricted using those names in `set_model_monitoring_function`, make sense?

Reply:
Yes, it sounds ok. Thanks